Dockerfile.installer.art

```diff
@@ -77,6 +77,8 @@ FROM registry.ci.openshift.org/ocp/4.18:base-rhel9
 
 RUN yum install --setopt=tsflags=nodocs -y jq && yum clean all && rm -rf /var/cache/yum/*
 
+RUN yum install --setopt=tsflags=nodocs -y jq && yum clean all && rm -rf /var/cache/yum/*
```

Comment:
that seems duplicated with the above line, is that correct?

Reply:
fixed 👍🏽